client/composables/fileTable/useFilesTable.ts

```diff
@@ -0,0 +1,18 @@
+import type { ComputedRef, Ref } from 'vue'
+
+export interface FilesTableContext {
+  selectedRow: Readonly<Ref<Nullable<FileRepresentationDTO>>>
+  selectedRowId: ComputedRef<string | undefined>
+  updateSelectedRow: (newRow: FileRepresentationDTO) => void
+  removeSelectedRow: () => void
+  unselectRow: () => void
+}
+
+export default function useFilesTable() {
+  const context = inject<FilesTableContext>(fileTableKey)!
```

Comment:
accidental bang, making the subsequent check useless. You should remove it

```suggestion
  const context = inject<FilesTableContext>(fileTableKey)
```